addon/components/detail.js

```diff
@@ -218,6 +218,17 @@ export default Component.extend(SpreadMixin, HookMixin, PropTypeMixin, {
     return propValidationResult && !_.isEmpty(propValidationResult.errors)
   },
 
+  @readOnly
+  @computed('selectedTabLabel')
```

Comment:
I think you misunderstood what I was asking here. This shouldn't be a computed property but rather an update to the action found [here](https://github.com/ciena-frost/ember-frost-bunsen/blob/master/addon/components/detail.js#L559-L565). It should look something like this:

```js
    /**
     * Change selected tab/root cell
     * @param {Number} tabIndex - index of root cell corresponding to tab
     */
    handleTabChange (tabIndex) {
      const onTabChange = this.get('onTabChange')
      const selectedTabCell = this.get(`precomputedCellConfig.${tabIndex}`)
      const tabAlias = getAlias(selectedTabCell)

      this.set('selectedTabIndex', tabIndex)

      if (onTabChange) {
        onTabChange(tabAlias, tabIndex)
      }
    },
```

You'll also need to make sure to update the [propTypes](https://github.com/ciena-frost/ember-frost-bunsen/blob/master/addon/components/detail.js#L93-L117) to include `onTabChange: PropTypes.func`